### PR TITLE
Fix #48

### LINKED ISSUE #48: Can't destroy the core after the fifth
- Core-version: `24` 
## Describe
Can't destroy the core after the fifth(the original position that haven't sorted by frequency)
The core list will refresh brfore destroy. Because cores are loaded five by five, the cores after the fifth won't be found instantly so Alicorn thinks the core was destroyed.(So she will fresh the core list again),making confirm window only flash by.

In addition,clicking the destroy will clear the statistic of the core, even if you cancel it

May caused by a9fa3129af5d6392c666dbbc84af0fba01bde760
## Problem recurrence
Create a container with more than five cores
(PS.Only need more than five cores, even if they come from different containers)
destroy a core after the fifth

### PR DESCRIPTION
**Pull request by iamyou**
Fix #48

### CODE REVIEW

**Comment by iamyou** [comment]
Fix an issue that core still remain after destroy.
(The change was rolled back)

File: src/renderer/LaunchPad.tsx [hunk 1]
```diff
@@ -296,14 +296,11 @@ function SingleCoreDisplay(props: {
...
             if (toDestroy) {
-              try{
-                await remove(
-                    getContainer(props.profile.container).getVersionRoot(toDestroy)
-                );
-              } finally {
-                markUsed(hash, 0);
-                props.refresh();
-              }
+              removeSync(
```
Comment (Andy-K-Sparklight) [inline src/renderer/LaunchPad.tsx]: Do not use sync method!

Comment (iamyou) [reply src/renderer/LaunchPad.tsx]: The change was rolled back.

Comment (iamyou) [reply src/renderer/LaunchPad.tsx]: Now it only fixes #48

**Review by Andy-K-Sparklight — APPROVED** [review]

**Pull request merged**